tests/Functional/src/com/microsoft/aad/adal/test/DefaultTokenCacheStoreTests.java

```diff
@@ -250,4 +264,116 @@ public void testExpireBuffer() {
     protected ITokenCacheStore getTokenCacheStore() {
         return new DefaultTokenCacheStore(this.getInstrumentation().getTargetContext());
     }
+    
+    private void addFRTCacheItem(DefaultTokenCacheStore defaultTokenCacheStore) {
+        final String TEST_AUTHORITY2 = "https://Developer.AndroiD.com/reference/android";
+        UserInfo user2 = new UserInfo("userid2", "givenName", "familyName", "identity", "userid2");
+        TokenCacheItem testFamilyRefreshTokenItemUser2 = new TokenCacheItem();
+        testFamilyRefreshTokenItemUser2.setAccessToken("user2token2Family");
+        testFamilyRefreshTokenItemUser2.setIsMultiResourceRefreshToken(true);
+        testFamilyRefreshTokenItemUser2.setAuthority(TEST_AUTHORITY2);
+        testFamilyRefreshTokenItemUser2.setUserInfo(user2);
+        testFamilyRefreshTokenItemUser2.setFamilyClientId("1");
+        testFamilyRefreshTokenItemUser2.setRefreshToken("user2FRT");
+        testFamilyRefreshTokenItemUser2.setClientId("");
+        testFamilyRefreshTokenItemUser2.setResource("");
+        defaultTokenCacheStore.setItem(CacheKey.createCacheKey(testFamilyRefreshTokenItemUser2),
+                testFamilyRefreshTokenItemUser2);
+    }
+    
+    public void testGetFamilyRefreshTokenItemForUser_withFRTCacheItem() throws IllegalArgumentException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException{
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
```

Comment:
This method is a private method used by the serialize() function and users do not have the access to this method.